app/src/main/java/fr/free/nrw/commons/media/MediaDetailInterface.java

```diff
@@ -13,6 +11,6 @@
      *
      * @param filename name of the file to be used for fetching captions
      * */
-    @GET("w/api.php?action=wbgetentities&props=labels&format=json&languagefallback=1")
-    Observable<MwQueryResponse> fetchCaptionByFilename(@Query("language") String language, @Query("titles") String filename);
+    @GET("w/api.php?action=wbgetentities&props=labels&format=json&languagefallback=1&languages=en&sites=commonswiki")
```

Comment:
Please add this comment:

"Please note that `languages=en` does not have an impact on the languages returned. All captions are returned for all languages."

Reply:
done